MetaMask/metamask-extension
# feat: UX: Multichain: Release the send page

## **Description**

Removes the feature flag surrounding the new send flow, as well as removes legacy send files.

## **Related issues**

N/A

## **Manual testing steps**

1. Go to Send page
2. See the new UI
3. Complete a send
4. Go to an NFT page
5. Click send
6. Send the NFT appropriately

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
